pkg/kubelet/cm/node_container_manager_linux.go

```diff
@@ -152,13 +155,7 @@ func enforceExistingCgroup(cgroupManager CgroupManager, cName CgroupName, rl v1.
 		Name:               cName,
 		ResourceParameters: rp,
 	}
-	if cgroupConfig.ResourceParameters == nil {
-		return fmt.Errorf("%q cgroup is not config properly", cgroupConfig.Name)
-	}
 	klog.V(4).InfoS("Enforcing limits on cgroup", "cgroupName", cName, "cpuShares", cgroupConfig.ResourceParameters.CpuShares, "memory", cgroupConfig.ResourceParameters.Memory, "pidsLimit", cgroupConfig.ResourceParameters.PidsLimit)
-	if !cgroupManager.Exists(cgroupConfig.Name) {
```

Comment:
Hmm. Not 100% sure about this one. The `cgroupManager.Exists` will emit all the missing control groups / controllers, while I assume the runc `Set` will not, and fail on the first one. Not sure if it important tho., but I assume we can take it in a separate PR and take a deep dive.

Reply:
Right, `cgroupManager.Exists` will report all missing cgroups on error, while a runc cgroup manager will only report the first missing cgroup. I'm not sure if there is a difference practically, since
 - normally they all either exist or not;
 - this code is assuming they all exist.
 
 If there is a need for detailed info, this call can be moved to error path (i.e. call `Exists` when `Update` failed).
 
 I'll rework it and move out of here.

Reply:
Brought the Exists check back.